platform-sdk/swirlds-logging/build.gradle.kts

```diff
@@ -38,3 +33,22 @@ testModuleInfo {
     requires("com.swirlds.common.test.fixtures")
     requires("jakarta.inject")
 }
+
+jmhModuleInfo {
+    requires("com.swirlds.logging")
+    requires("org.apache.logging.log4j")
+    requires("com.swirlds.config.api")
+    runtimeOnly("com.swirlds.config.impl")
+    requires("org.apache.logging.log4j.core")
+    requires("com.github.spotbugs.annotations")
+    requires("jmh.core")
+}
+
+// IntelliJ uses adhoc-created JavaExec tasks when running a 'main()' method.
```

Comment:
@jjohannes @nathanklick looks like we need that block whenever we want to exec a main. Can we add that to a plugin?